### PR TITLE
fix: image link in readme.md

### PR DESCRIPTION
**Pull request by sreyasaju**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,5 +2,12 @@
 
 [![Sugaroid Bot](https://img.shields.io/badge/sugaroid-discord-%23f70049)](https://github.com/sugaroidbot/sugaroid)
 
+Sugaroid also has a discord instance running on a ``Heroku`` server.
```
Comment (srevinsaju) [inline README.md]: Why 'also'? 

When you are copying something, make sure you have perfection

File: README.md [hunk 2]
```diff
@@ -2,5 +2,12 @@
...
+Sugaroid Discord bot is powered by Python Async IO, to get faster
+results and ``discord.py``. The discord bot is (somewhat) frequently
```
Comment (srevinsaju) [inline README.md]: This part uses rst, not markdown. Convert it into markdown

File: README.md [hunk 3]
```diff
@@ -2,5 +2,12 @@
...
+tested by developers to find it out bugs. The message from the bot is
+demojized to remove static unicode emoji’s and then converted into
+``emocodes`` which is better than unicode emoji’s on Discord.
```
Comment (srevinsaju) [inline README.md]: Sugaroid doesn't use emocodes anymore.

**Review by srevinsaju — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sreyasaju) [reply README.md]: Oops! sorry for that 😁

File: README.md [hunk 4]
```diff
@@ -2,5 +2,7 @@
 
 [![Sugaroid Bot](https://img.shields.io/badge/sugaroid-discord-%23f70049)](https://github.com/sugaroidbot/sugaroid)
 
+Discord wrapper for the Sugaroid Bot.
+
+![Sugaroid Discord Bot at work](https://g.sugaroid.srevinsaju.me/docs/_images/sugaroid_discord.png)
 
-![Sugaroid Discord Bot at work](http://g.sugaroid.srevinsaju.me/docs/img/sugaroid_discord.png)
```
Comment (srevinsaju) [inline README.md:6]: Why do you want to force use a non secure link. i.e `http`?

**Review by srevinsaju — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (sreyasaju) [reply README.md:6]: @srevinsaju I guess you've mixed up with my additions and deletions

Comment (sreyasaju) [reply README.md:6]:
```
Discord wrapper for the Sugaroid Bot.

![Sugaroid Discord Bot at work](https://g.sugaroid.srevinsaju.me/docs/_images/sugaroid_discord.png)
```

This is what I've added

**Review by srevinsaju — APPROVED** [review]

**Pull request merged**